src/components/containers/Fold.js

```diff
@@ -4,21 +4,44 @@ import classnames from 'classnames';
 import {CloseIcon, AngleDownIcon} from 'plotly-icons';
 
 export default class Fold extends Component {
-  constructor() {
-    super();
-    this.state = {folded: false};
+  constructor(props, context) {
+    super(props, context);
+    const {individualFoldStates} = context;
+    if (!individualFoldStates) {
```

Comment:
Could we just make `individualFoldStates` required and stop supporting both modes? Is there a use-case for a Fold that's not in a Panel like this?

Reply:
There's not a use case yet, just thought it might have been useful to not have this coupled.

Reply:
In that case my (weak) preference would be to couple them strongly by making the props and contexts required, so as to avoid people occasionally forgetting to add them everywhere etc. YAGNI :)

Reply:
Yeah, I think you're right, I'll do that